library/src/containers/AsyncApi/Standalone.tsx

```diff
@@ -32,7 +32,6 @@ class AsyncApiComponent extends Component<AsyncApiProps, AsyncAPIState> {
     if (parsedSpec) {
       this.state = { asyncapi: parsedSpec };
     }
-    return;
```

Comment:
please revert this change,
https://github.com/asyncapi/asyncapi-react/pull/802 PR handles that change

Reply:
done